### PR TITLE
:ambulance: TypeError: this.$buttons.find is not a function

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: project_task_subtask/doc/changelog.rst [hunk 1]
```diff
@@ -1,3 +1,8 @@
+`1.1.1`
+-------
+
+**Fix:** Compatibility issue - error on opening a task
```
Comment (yelizariev) [inline project_task_subtask/doc/changelog.rst]: Describe in changelog  which compatibilitiy you mean

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Review by em230418 — APPROVED** [review]
Кроме упомянутого https://github.com/it-projects-llc/misc-addons/pull/868#discussion_r353250004 предложений нет

File: project_task_subtask/static/src/js/one2many_renderer.js [hunk 2]
```diff
@@ -95,7 +95,10 @@ odoo.define('project_task_subtask.one2many_renderer', function(require){
         _renderControlPanel: function () {
             var self = this;
             if (this.check_task_tree_mode()) {
-                this.$buttons = QWeb.render("SubtaskSortButtons", {});
+                if (!this.$buttons) {
+                    this.$buttons = $('<div/>');
```
Comment (yelizariev) [inline project_task_subtask/static/src/js/one2many_renderer.js:99]: Add a comment that it's about compatibility with multi_attachment_product_image

Comment (KolushovAlexandr) [reply project_task_subtask/static/src/js/one2many_renderer.js:99]: @yelizariev it's not so much about the compatibility with exact module how about proper inheritance in general. I changed the changelog so there is no notice about compatibility.

**Review by yelizariev — APPROVED** [review]

**Comment by em230418** [comment]
I approve to merge it now

**Pull request merged**